lib/engine/game/g_1848/game.rb

```diff
@@ -369,6 +387,15 @@ def new_auction_round
           ])
         end
 
+        def stock_round
+          Engine::Round::Stock.new(self, [
+            Engine::Step::DiscardTrain,
+            Engine::Step::Exchange,
+            Engine::Step::SpecialTrack,
```

Comment:
Also, is there a case for laying track in the SR?

Reply:
Found it in the rules.

Reply:
where?